.github/workflows/docs-validation.yml

```
@@ -161,10 +163,86 @@ jobs:
+            instance = json.load(f)
+
+        try:
+            jsonschema.validate(instance=instance, schema=schema)
+            sys.exit(0)
+        except jsonschema.ValidationError as e:
+            print(f"Validation error: {e.message}", file=sys.stderr)
+            sys.exit(1)
+        PYEOF
+
+    - name: Validate curriculum JSON against UMCF schema
+      run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2016:info:17:14: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)